enterprise/hd-memory-client-server/hd-memory-examples-server/src/main/java/MemoryStatsUtil.java

```diff
@@ -0,0 +1,26 @@
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.instance.HazelcastInstanceImpl;
+import com.hazelcast.instance.HazelcastInstanceProxy;
+import com.hazelcast.memory.DefaultMemoryStats;
+import com.hazelcast.memory.MemoryStats;
+import com.hazelcast.util.ExceptionUtil;
+
+import java.lang.reflect.Method;
+
+public class MemoryStatsUtil {
+
+    public static MemoryStats getMemoryStats(HazelcastInstance hz) {
```

Comment:
Are you sure this gives you right Memory stats? (I mean HD memory stats)

I think you should also have a look at this
``` java
Node node = TestUtil.getNode(instance); // or another way for getting "Node" over "HazelcastInstance"
EnterpriseSerializationService serializationService =
                (EnterpriseSerializationService) node.getSerializationService();
MemoryManager memoryManager = serializationService.getMemoryManager();
MemoryStats memoryStats = memoryManager.getMemoryStats();
```